plugins/Feedback/templates/index.twig

```diff
@@ -35,13 +35,11 @@
             <p>{{ 'Feedback_ProfessionalServicesIntro'|translate }}</p>
 
             <p>{{ 'Feedback_ProfessionalServicesOfferIntro'|translate }}</p>
-            <p> &bull; {{ 'Feedback_ProfessionalServicesReviewPiwikSetup'|translate }}</p>
-            <p> &bull; {{ 'Feedback_ProfessionalServicesOptimizationMaintenance'|translate }}</p>
-            <p> &bull; {{ 'Feedback_ProfessionalServicesPhoneEmailSupport'|translate }}</p>
+            <p> &bull; {{ 'Feedback_ProfessionalServicesDedicatedSupport'|translate }}</p>
+            <p> &bull; {{ 'Feedback_ProfessionalServicesOnboarding'|translate }}</p>
             <p> &bull; {{ 'Feedback_ProfessionalServicesTraining'|translate }}</p>
-            <p> &bull; <a  rel='noreferrer noopener' target='_blank' href='https://matomo.org/recommends/premium-plugins'>{{ 'Feedback_ProfessionalServicesPremiumFeatures'|translate }}</a></p>
-            <p> &bull; {{ 'Feedback_ProfessionalServicesCustomDevelopment'|translate }}</p>
-            <p> &bull; {{ 'Feedback_ProfessionalServicesAnalystConsulting'|translate }}</p>
+            <p> &bull; {{ 'Feedback_ProfessionalServicesSupport'|translate }}</p>
+            <p> &bull; {{ 'Feedback_ProfessionalServicesEmailAlerts'|translate }}</p>
 
             <form target="_blank" action="https://matomo.org/support/">
```

Comment:
Can we link here to the support plans @katebutler  as discussed on the call? https://matomo.org/support-plans/